medium/foundation.md

```diff
@@ -0,0 +1,16 @@
+* **io.js és a node.js Foundation**
+
+Az egyeztetés jelenlegi állapota.
+
+Egy héttel ezelőtt, Scott Hammond a Joyent CEO, meghivta egy személyes találkozóra az io.js (Technical Committee) TC csapatát, ahol feltárta elképzeléseit, miként is vezetné vissza az io.js-t a node.js projectbe.
```

Comment:
*miként is vezetné vissza az io.js-t a node.js projectbe* -> *hogyan egyesítené az io.js-t és a node.js-t*